include/pfs/drp/stella/SpectraTemplates.hpp

```diff
@@ -524,10 +523,16 @@ bool pfs::drp::stella::Spectrum<SpectrumT, MaskT, VarianceT, WavelengthT>::ident
     /// Check for monotonic
     if ( math::isMonotonic( _wavelength ) == 0 ){
       cout << "Identify: WARNING: Wavelength solution is not monotonic => Setting identifyResult.rms to 1000" << endl;
+      cout << "Identify: _wavelength = " << _wavelength << endl;
       _dispRms = 1000.;
       LOGLS_WARN(_log, "Identify: RMS = " << _dispRms);
       LOGLS_WARN(_log, "======================================");
     }
+    else{
+      #ifdef __DEBUG_IDENTIFY__
+        cout << "identify: _wavelength is monotonic " << endl;
```

Comment:
Aren't we moving to using logging for this? Why are we adding more references to `cout`?